README.md

```diff
@@ -1,2 +1,46 @@
 # container-ipam-tool
 Configuration tool for any container environments
+
+## Prerequisite
+
+To ctreate the EA(Extensible Attributes) definition, you need access to the Infoblox DDI product. For evaluation purposes, you can download a
+virtual version of the product from the Infoblox Download Center (https://www.infoblox.com/infoblox-download-center)
+Alternatively, if you are an existing Infoblox customer, you can download it from the support site.
+
+## Configuring Cloud Extensible Attributes using create-ea-defs tool
+
+If the "Cloud Network Automation" license is activated, then the Cloud Extensible Attributes used by the ipam-plugin
+can be defined using the create-ea-defs tool in the infoblox/container-ipam-tools docker image.
```

Comment:
If the "Cloud Network Automation" license is activated, then the Cloud Extensible Attributes used by the ipam-plugin can be created using the infoblox/container-ipam-tools docker image. There are 2 ways to do it